tests/Stubs/TestModel.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Spatie\Translatable\Test;
+namespace Spatie\Translatable\Test\Stubs;
```

Comment:
I'd prefer the `TestModel` to be in the `Spatie\Translatable\Test` namespaces. Please revert this change.